Database/schema.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS price_readings, subscriptions, users, products;
+
+CREATE TABLE products(
+    product_id SERIAL PRIMARY KEY,
```

Comment:
Why did you use serial? This is an older version. Try to use default or default(1,1)